rfcs/RFC-Upload-From-Dropbox.md

```diff
@@ -0,0 +1,358 @@
+# User Uploads From Drop Box
```

Comment:
I know we discussed different RFC for Dropbox and Google Drive.  But I think this RFC and the one for google is probably going to have >98% overlap.  
Maybe it would be worth combining those to, and having a section in the Upload Server that says  something like 
'For DropBox upload we will use <name of package or API>."
'For Google Drive we will use <name of package or API>".
Nothing outside that one component will have to know the difference.

Reply:
Well, actually how will the Upload Component know if the link is for dropbox or google drive.  Will it parse the sharable link to deduce that.   Or do we plan to have something explicit in the user interface where the user has some radial button that said "DropBox" or "Google Drive", and has to select one (along with the link), and then the selection is plumbed all the way through the API to the Download Queue Entry to the Upload Server.

Reply:
I think parsing the link would be the easiest way. The URL needs to parse either way to build the actual download URL.

Reply:
We can amend this RFC later to be both dropbox and google drive.

Reply:
We can amend this document later to include google drive.